### PR TITLE
Update modules

### PR DESCRIPTION
**Pull request by JoseEspinosa**
Bump modules versions

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Make sure your code lints (`nf-core lint .`).
- [x] `CHANGELOG.md` is updated

### CODE REVIEW

File: subworkflows/local/prepare_genome.nf [hunk 1]
```diff
@@ -130,7 +130,7 @@ workflow PREPARE_GENOME {
     if (prepare_tool_index == 'bwa') {
         if (params.bwa_index) {
             if (params.bwa_index.endsWith('.tar.gz')) {
-                ch_bwa_index = UNTAR_BWA_INDEX ( params.bwa_index ).untar
+                ch_bwa_index = UNTAR_BWA_INDEX ( [], params.bwa_index ).untar.map{it -> it[1]}
```
Comment (drpatelh) [inline subworkflows/local/prepare_genome.nf:133]:
```suggestion
                ch_bwa_index = UNTAR_BWA_INDEX ( [], params.bwa_index ).untar.map{ it[1] }
```

File: subworkflows/local/prepare_genome.nf [hunk 2]
```diff
@@ -148,7 +148,7 @@ workflow PREPARE_GENOME {
     if (prepare_tool_index == 'bowtie2') {
         if (params.bowtie2_index) {
             if (params.bowtie2_index.endsWith('.tar.gz')) {
-                ch_bowtie2_index = UNTAR_BOWTIE2_INDEX ( params.bowtie2_index ).untar
+                ch_bowtie2_index = UNTAR_BOWTIE2_INDEX ( [], params.bowtie2_index ).untar.map{it -> it[1]}
```
Comment (drpatelh) [inline subworkflows/local/prepare_genome.nf:151]:
```suggestion
                ch_bowtie2_index = UNTAR_BOWTIE2_INDEX ( [], params.bowtie2_index ).untar.map{ it[1] }
```

File: subworkflows/local/prepare_genome.nf [hunk 3]
```diff
@@ -184,7 +184,7 @@ workflow PREPARE_GENOME {
     if (prepare_tool_index == 'star') {
         if (params.star_index) {
             if (params.star_index.endsWith('.tar.gz')) {
-                ch_star_index = UNTAR_STAR_INDEX ( params.star_index ).untar
+                ch_star_index = UNTAR_STAR_INDEX ( [], params.star_index ).untar.map{it -> it[1]}
```
Comment (drpatelh) [inline subworkflows/local/prepare_genome.nf:187]:
```suggestion
                ch_star_index = UNTAR_STAR_INDEX ( [], params.star_index ).untar.map{ it[1] }
```

**Review by drpatelh — COMMENTED** [review]
You will need to update all calls to `GUNZIP` in the pipeline to use a `meta` map too like you have done with `UNTAR`.  e.g. [here](https://github.com/nf-core/rnaseq/blob/43e2ef355428e83a63d0642b7d38e119f6f23e75/subworkflows/local/prepare_genome.nf#L46)

File: subworkflows/local/prepare_genome.nf (hunk 1 above)
Comment (JoseEspinosa) [reply subworkflows/local/prepare_genome.nf]: Addressed in [33d3bdd](https://github.com/nf-core/chipseq/pull/271/commits/33d3bdda056cc66bd6c9070bc0abf670e9c6c229)

File: subworkflows/local/prepare_genome.nf (hunk 2 above)
Comment (JoseEspinosa) [reply subworkflows/local/prepare_genome.nf]: Addressed in [33d3bdd](https://github.com/nf-core/chipseq/pull/271/commits/33d3bdda056cc66bd6c9070bc0abf670e9c6c229)

File: subworkflows/local/prepare_genome.nf (hunk 3 above)
Comment (JoseEspinosa) [reply subworkflows/local/prepare_genome.nf]: Addressed in [33d3bdd](https://github.com/nf-core/chipseq/pull/271/commits/33d3bdda056cc66bd6c9070bc0abf670e9c6c229)

**Comment by JoseEspinosa** [comment]
Very nice spot @drpatelh :smile: Did not realised :cold_sweat:

**Review by drpatelh — APPROVED** [review]
🚀

**Pull request merged**